taprpc/rfqrpc/rfq.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+
+package rfqrpc;
+
+option go_package = "github.com/lightninglabs/taproot-assets/taprpc/rfqrpc";
+
+service Rfq {
+    /* tapcli: `rfq buyorder`
+    UpsertAssetBuyOrder is used to upsert a buy order for a specific asset.
+    */
+    rpc UpsertAssetBuyOrder (UpsertAssetBuyOrderRequest)
+        returns (UpsertAssetBuyOrderResponse);
+
+    /* tapcli: `rfq selloffer`
+    UpsertAssetSellOrder is used to upsert a sell offer for a specific asset.
+     */
```

Comment:
micro-nit: indentation of closing comment is off by one space.

Reply:
And on the other comments also, thanks :+1: